server/app.js

```diff
@@ -224,12 +224,12 @@ const createApp = ({
   }
 
   // Routing
+  if (!process.env.HOTJAR_ID) {
```

Comment:
this was a bit noisy - so moved to just on start up

Reply:
🙌 would it be worth sending this warning to Sentry as well?

Reply:
added capturing a message in sentry. 

Docs say that these are not usually emitted. not sure if it's specific config we need to enable